libs/wallet-ui/src/components/code-window/code-window.spec.tsx

```diff
@@ -0,0 +1,26 @@
+import { render, screen } from '@testing-library/react'
+import { CodeWindow } from './code-window'
+
+jest.mock('../copy-with-tooltip', () => ({
+  CopyWithTooltip: () => <div data-testid="copy-button" />,
+}))
+
+const renderComponent = () => {
+  return render(
+    <CodeWindow
```

Comment:
**praise (non-blocking)**: I think the raw JSON view is more useful here than adopting / adapting components from the Block Explorer - the protos are generally pretty succinct.